app/api/v2/managers/operation_api_manager.py

```diff
@@ -20,3 +48,60 @@ async def get_operation_object(self, operation_id: str, access: dict):
         if operation.match(access):
             return operation
         raise JsonHttpForbidden(f'Cannot view operation due to insufficient permissions: {operation_id}')
+
+    async def setup_operation(self, data: dict, access: BaseWorld.Access):
+        """Applies default settings to an operation if data is missing."""
+        planner_name = data.pop('planner', {}).get('name', '')
+        planner = await self._construct_planner(planner_name)
+        adversary_data = data.pop('adversary', {})
+        adversary_id = adversary_data.get('adversary_id', '')
+        adversary = await self._construct_adversary(adversary_id)
+        group_data = data.pop('host_group', '')
```

Comment:
I'm wondering if it's better to just have the user specify a group name, or leave this field out if they want all groups used (kind of like how we currently do it through the GUI), That way they don't have to build the list of all the agent paws they want, which can be cumbersome - we want to maintain the current functionality of "select group to pick which agents you want involved".

The operation object does have a `group` variable, which we could add to the schema to allow users to set this value. We do need to establish the group value so that agents are added appropriately when `update_operation` is called from various services.